examples/vnet/k8s-vnet-azure-cni-predeploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This file is basically the same as the existing kubenet VNET predeploy script, but we create a custom VNET with only one address space to more accurately match the current functional support for k8s w/ Azure CNI (documentation explains it above, but the short story is that we don't have a 100% resilient solution yet for custom VNETs w/ lots of address space that can't be rationalized into a single CIDR network block).